### ISSUE TITLE
linreg(::Array{Float64,1}, ::Array{Float64,1}, ::Array{Float64,1})

### ISSUE THREAD

**Issue by LeoK987** [JuliaLang/julia#18349]
```jl
  | | |_| | | | (_| |  |  Version 0.5.0-rc3+0 (2016-08-22 23:43 UTC)
 _/ |\__'_|_|_|\__'_|  |  Official http://julialang.org/ release
|__/                   |  x86_64-unknown-linux-gnu

MethodError: no method matching linreg(::Array{Float64,1}, ::Array{Float64,1}, ::Array{Float64,1})

julia> methods(linreg)
# 1 method for generic function "linreg":
linreg(x::AbstractArray{T<:Any,1}, y::AbstractArray{T<:Any,1}) at linalg/generic.jl:463
```

**Comment by yuyichao** [comment]
Please be more specific to what the issue is about.

How did you get the error?
What's the backtrace?

This is very likely a user/package error since AFAIK the function is never called with 3 arguments in `Base`.

**Comment by LeoK987** [comment]
A direct call with linreg(x, y, w) gets this error.

The 3-argument version of linreg is in version 0.4.  In 0.5, the documentation still references it, but the code obviously was not implemented.

**Comment by yuyichao** [comment]
The weighted form is intentionally removed in https://github.com/JuliaLang/julia/pull/16545 Where did you find the reference from 0.5 doc?

**Comment by LeoK987** [comment]
It is in the version of the 0.5 doc I downloaded probably a couple weeks ago.  

> linreg(x, y, w)
> Weighted least-squares linear regression.

I don't know how get the most updated version of 0.5 doc now and so can not say that it is there. But I did find that it is not referenced in the 0.6 doc.

**Comment by yuyichao** [comment]
How many weeks and where and which version did you download. AFAICT the binary download is up to date.

**Comment by LeoK987** [comment]
OK.  Is that for 0.5?  It appears to me the latest doc now is 0.6 and the stable is 0.4, so I don't where to get 0.5 doc now.  The 0.5 doc I got is within a month.

**Comment by LeoK987** [comment]
I get my docs here only: http://docs.julialang.org/en/release-0.4/

**Comment by andreasnoack** [comment]
The 0.5 docs will probably only be available when 0.5 is released but the latest documents should be very close to how the 0.5 docs will look like. Do you still think there is an issue here?

**Comment by LeoK987** [comment]
In that case I am fine.
